src/nss/on_mod_load.nss

```diff
@@ -55,6 +55,49 @@ void SpawnPCBloodstains()
     }
 }
 
+void EnsureAreaBilateralLinkages()
```

Comment:
Hey does this mean that it only needs to be set on one area? If so do I need to clean up the other linked area variables?